.github/workflows/ci.yml

```diff
@@ -10,7 +10,7 @@ name: Continuous Integration
 on:
   pull_request:
     branches: ['**']
-    types: [opened, synchronize, reopened, edited]
+    types: [opened, synchronize, reopened, edited, labeled, unlabeled]
```

Comment:
this will skip the label doc but it will rerun the Benchmarks whenever any label is added or removed

Reply:
I added this so that if we remove the documentation label then again CI runs and run workflow accordingly